<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Identification</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>Model</string>
    </property>
    <widget class="QComboBox" name="comboBoxModel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>231</width>
       <height>37</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Model type</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>351</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string>Model Setup</string>
     </property>
     <widget class="QLabel" name="labelModel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>378</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>331</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayoutModelSetup"/>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Samples</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxSamples">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>26</y>
       <width>221</width>
       <height>42</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>431</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Results</string>
    </property>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>430</y>
       <width>103</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollAreaResults">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>341</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>387</width>
        <height>337</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>481</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Plots</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>451</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutPlot"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
